Samples/BackgroundBlur/BackgroundBlur/SegmentModel.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+#include <winrt/Microsoft.AI.MachineLearning.Experimental.h>
+#include <winrt/Microsoft.AI.MachineLearning.h>
+#include <Windows.AI.MachineLearning.native.h>
+#include <strsafe.h>
+#include <wtypes.h>
+#include <winrt/base.h>
+
+using namespace winrt::Microsoft::AI::MachineLearning;
+using namespace winrt::Microsoft::AI::MachineLearning::Experimental;
```

Comment:
Avoid `using namespace` in headers, it pollutes the namespace of any file that includes this header

Reply:
(As a best practice avoid it in general, but in `cpp` files we tend to use it elsewhere)

Reply:
Makes sense, I can at least move it to SegmentModel.cpp